assets/js/translator.js

```diff
@@ -1104,6 +1154,11 @@ function muteLanguages() {
             $(this).addClass('text-muted');
         }
     });
+    $.each($('#dstLanguages .languageVariant'), function () {
```

Comment:
Elements with the `languageVariant` class also have the `languageName` class, so the lines just above this already deal with them, right?